Compute/src/V1/Gapic/AcceleratorTypesGapicClient.php

```diff
@@ -51,29 +54,24 @@
  * ```
  * $acceleratorTypesClient = new AcceleratorTypesClient();
  * try {
- *     $project = '';
+ *     $project = 'project';
  *     // Iterate over pages of elements
  *     $pagedResponse = $acceleratorTypesClient->aggregatedList($project);
  *     foreach ($pagedResponse->iteratePages() as $page) {
- *         foreach ($page as $key => $element) {
```

Comment:
Aggregated lists have a `$key` element. The monolith had logic to determine if the iterator was for a map or a list, and used this syntax for maps (since `$key` in this case contains some valuable identifier)

Reply:
Done, and [fixed](https://github.com/googleapis/gapic-generator-php/pull/284) in the microgenerator.

Reply:
However, note that this particular instance will stay the same, and that's working as intended. The pagination design doc says that `repeated` fields take precedence over `map` fields (see below). Therefore, the resources for this RPC is `unreachables`, not `items`, so this `foreach` is fine as-is.

CC @vam-google 

_If the response message has only one repeated field treat it as the one with paginated resources (i.e. ignore the map<string, ?> fields if any)._